rpm/SPECS/docker-ce-rootless-extras.spec

```diff
@@ -0,0 +1,64 @@
+%global debug_package %{nil}
+
+Name: docker-ce-rootless-extras
+Version: %{_version}
+Release: %{_release}%{?dist}
+Epoch: 0
+Source0: engine.tgz
+Summary: Rootless support for Docker
+Group: Tools/Docker
+License: ASL 2.0
+URL: https://docs.docker.com/engine/security/rootless/
+Vendor: Docker
+Packager: Docker <support@docker.com>
+
+Requires: docker-ce
+# slirp4netns >= 0.4 is available in the all supported versions of CentOS and Fedora.
+Requires: slirp4netns >= 0.4
+# fuse-overlayfs >= 0.7 is available in the all supported versions of CentOS and Fedora.
+Requires: fuse-overlayfs >= 0.7
```

Comment:
Wanted to know how much extra this would install, but it looks limited:

```
==========================================================================================================================================================================================
 Package                                       Arch                                     Version                                            Repository                                Size
==========================================================================================================================================================================================
Installing:
 slirp4netns                                   x86_64                                   0.4.3-4.el7_8                                      extras                                    81 k
Installing for dependencies:
 libseccomp                                    x86_64                                   2.3.1-4.el7                                        base                                      56 k

Transaction Summary
==========================================================================================================================================================================================
Install  1 Package (+1 Dependent package)

Total download size: 138 k
Installed size: 466 k


==========================================================================================================================================================================================
 Package                                          Arch                                     Version                                         Repository                                Size
==========================================================================================================================================================================================
Installing:
 fuse-overlayfs                                   x86_64                                   0.7.2-6.el7_8                                   extras                                    54 k
Installing for dependencies:
 fuse3-libs                                       x86_64                                   3.6.1-4.el7                                     extras                                    82 k
Transaction Summary
==========================================================================================================================================================================================
Install  1 Package (+1 Dependent package)
Total download size: 136 k
Installed size: 386 k
```

Fairly small, so I think that should be fine